mod-source-record-storage-server/src/main/java/org/folio/errorhandlers/ParsedRecordChunksErrorHandler.java

```diff
@@ -57,7 +58,11 @@ public void handle(Throwable throwable, KafkaConsumerRecord<String, String> reco
     String jobExecutionId = okapiConnectionParams.getHeaders().get(JOB_EXECUTION_ID_HEADER);
     String tenantId = okapiConnectionParams.getTenantId();
 
-    sendErrorRecordsSavingEvents(recordCollection, throwable.getMessage(), kafkaHeaders, jobExecutionId, tenantId);
+    if(throwable instanceof DuplicateEventException) {
+      LOGGER.warn("Error parsing event for jobExecutionId: {} , tenantId: {}, cause: {}", jobExecutionId, tenantId, throwable.getMessage());
```

Comment:
The text of this log message should be changed to someone like this: "Duplicate event received, skipping processing" and include jobExecutionId, chunkId or recordId

Reply:
Thanks. I forgot to fix the temporary draft text. We can't add recordId because there can be multiple records. We don't have access to chunkNumber from ErrorHandler.

Reply:
hello Khamid, please add "correlationId" header to log message (okapiConnectionParams.getHeaders().get("correlationId")) to track what chunk/event was processed repeatedly

Reply:
> hello Khamid, please add "correlationId" header to log message (okapiConnectionParams.getHeaders().get("correlationId")) to track what chunk/event was processed repeatedly

Hello Ruslan. Thank you for comment. Done